drivers/unix/net_socket_posix.cpp

```diff
@@ -126,12 +126,12 @@ static size_t _set_addr_storage(struct sockaddr_storage *p_addr, const IP_Addres
 			addr4->sin_addr.s_addr = INADDR_ANY;
 		}
 
-		copymem(&addr4->sin_addr.s_addr, p_ip.get_ipv4(), 16);
+		copymem(&addr4->sin_addr.s_addr, p_ip.get_ipv4(), 4);
```

Comment:
I also found this small issue, which was copied over from `socket_helper.h` during the refactoring.
I don't think it deserves to be in its own commit.
While this looks bad, it's most likely harmless, the struct is actually a `sockaddr_storage` which can usually be seen as a union (although it isn't excatly, as per [RFC2553](https://tools.ietf.org/html/rfc2553)) so it won't write out of that var.
Still, this should be fixed :wink: